java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -73,4 +73,21 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> bitcount(String key, long start, long end, BitmapIndexType options);
+
+    /**
+     * Returns the bit value at <code>offset</code> in the string value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/getbit/">redis.io</a> for details.
+     * @param key The key for the string to get the bit at offset of.
+     * @param offset The index of the bit to return.
+     * @return The bit at offset of the string. Returns zero if the key is missing as it is treated as
+     *     an empty string. Returns zero if the positive offset exceeds the length of the string as it
+     *     is assumed to be padded zeroes.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.getbit("myKey1", 1).get();
+     * assert payload == 1L; // The second bit for string stored at "myKey1" is set to 1.
```

Comment:
second or first? according to your doc, it does not return 0 on success

Reply:
It is the second element as the second parameter is 1 which is the index. 0 or 1 does not determine success, it is just the possible value of the bit at the specified element.